docs/configuration_files.rst

```diff
@@ -7,92 +7,92 @@ Jwst_reffiles makes use of multiple configuration files to control the behavior
 
 .. _main_cfg:
 
-An example of the main configuraiton file:
+An example of the main configuration file:
 
-::
+.. parsed-literal::
 
-    instrument: NIRCam
-    reflabels: [example_readnoise_module]
-    reftypes: [bpm,rdnoise,gain,linearity]
+    instrument_: NIRCam
+    reflabels_: [example_readnoise_module]
+    reftypes_: [bpm,rdnoise,gain,linearity]
 
-    default_reflabels:
+    default_reflabels_:
        bpm: bpm
        rdnoise: example_readnoise_module
        gain: gain_armin
 
     inputfiles:
-       dark_pattern: Bias|Dark|BIAS|DARK|ZERO
-       flat_pattern: GAIN|LIN
+       dark_pattern_: Bias|Dark|BIAS|DARK|ZERO
```

Comment:
should bias be here twice?

Reply:
Yes. This is here so that both upper and mixed-case are used when searching for filenames. This will all have to change when dealing with real JWST data, where the filenames don't have any of these strings.